frontend/src/CreateForm.jsx

```diff
@@ -5,19 +5,21 @@ function CreateFrom(props) {
   const [user, setUser] = useState(
     {
       name: '',
-      pass: '',
+      email: '',
+      password: '',
     },
   );
 
   function handleChange(event) {
     const { name, value } = event.target;
-    if (name === 'pass') setUser({ name: user.name, pass: value });
-    else setUser({ name: value, pass: user.pass });
+    if (name === 'password') setUser({ name: user.name, email: user.email, password: value });
+    else if (name === 'name') setUser({ name: value, email: user.email, passowrd: user.password });
+    else setUser({ name: user.name, email: value, password: user.password });
```

Comment:
This seems a little redundant

Reply:
I think it's fine the way it is.

Reply:
The reason it's programmed this way is because the password, name, and email all have the same handler when they are changed. I need to check what value was being passed to me so I know what value to update in the user.

Reply:
Okay makes sense